Libs/MRML/Core/vtkMRMLSubjectHierarchyNode.cxx

```diff
@@ -2584,7 +2584,7 @@ void vtkMRMLSubjectHierarchyNode::RemoveAllItems(bool removeDataNode/*=false*/)
 }
 
 //----------------------------------------------------------------------------
-void vtkMRMLSubjectHierarchyNode::SetItemParent(vtkIdType itemID, vtkIdType parentItemID)
+void vtkMRMLSubjectHierarchyNode::SetItemParent(vtkIdType itemID, vtkIdType parentItemID, bool skipCircularCheck/*=false*/)
```

Comment:
bool skipCircularCheck=false => double-negative is a bit harder to decode

why not use an argument name like _safeMode_ or _enableCircularCheck_

Reply:
I didn't like the safeMode attribute when I saw it in Kyle's PR either, I think it's so generic it can be misunderstood. I'll call it enableCircularCheck

Reply:
OK!